ios/FluentUI/Command Bar/CommandBarCommandGroupsView.swift

```diff
@@ -135,4 +138,8 @@ class CommandBarCommandGroupsView: UIView {
         sender.item.handleTapped(sender)
         sender.updateState()
     }
+
+    private func usesEqualWidthGroups() -> Bool {
+        return buttonGroupsStackView.distribution == .fillEqually
```

Comment:
PR notes suggest we should be checking whether we're scrollable here to determine whether to use equal width groups? Is this just an indirect way of doing so?

Reply:
Removed in this in favor of the new property I added.